docs/pages/blog/material-ui-v6-sneak-peek.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Material UI v6 Sneak Peek
+description: Previewing all the major updates of Material UI v6.
+date: 2024-07-01T00:00:00.000Z
+authors: ['siriwatknp']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+Material UI v6 is now in beta! We're excited to share a sneak peek of all the upcoming major updates.
```

Comment:
Not sure if "sneak peek" is the right term because users can already try these features.

```suggestion
Material UI v6 is now in beta! We're excited to share all the upcoming major updates.
```